optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SolverProblemBenchmarkResult.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.benchmark.impl.result;
+
+import java.io.File;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.config.solver.SolverConfig;
+
+public interface SolverProblemBenchmarkResult {
```

Comment:
This SolverProblemBenchmarkResult should be renamed back to SingleBenchmarkResult (I presume it had this temporary name due to refactoring),
as well as all the fields/getters that were renamed (excluding those that needed to be renamed to SubSingle* of course).

Oh wait. This is an interface. Why has this interface been extracted? If we extract it here, why don't we extract it for the others too? If we want to keep the interface, then this interface should be named SingleBenchmarkResult and it's implemtation DefaultSingleBenchmarkResult.
But do we want to keep this interface?